src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -103,14 +107,16 @@ public String getOgcCollection(@RequestParam(value = "f", required = false, defa
 			},
 			externalDocs=@ExternalDocumentation(url="http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_feature_")
 		)
+	@Parameter(name="bbox", in=ParameterIn.QUERY, description=BBOX_DESCRIPTION)
 	@GetMapping(value = "collections/{collectionId}/items", produces = MediaType.APPLICATION_JSON_VALUE)
 	public String getOgcCollectionFeatures(
 			@RequestParam(value = "f", required = false, defaultValue = "json") String mimeType,
 			@Min(value=1, message = LIMIT_VALIDATE_MESS) @RequestParam(value = "limit", required = false, defaultValue = "10000") int limit,
 			@Min(value=0, message = START_INDEX_VALIDATE_MESS) @RequestParam(value = "startIndex", required = false, defaultValue = "0") int startIndex,
-			@BBox @RequestParam(value = "bbox", required = false) BoundingBox bbox,
+			@BBox @RequestParam(value = "bbox", required = false) String bbox,
```

Comment:
I am curious if Spring has a List<String> option where it automatically splits on the comma for us. It feels like I remember such a thing but it might just be wishful thinking.

Reply:
The default is to split on the comma and create an array. In the case of bbox, that is not necessarily what we want.

Reply:
Spring does have a list option. This parameter was changed so the swagger document
UI says the type of the bbox parameter is a String.